bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -35,6 +35,7 @@
 {% endblock %}
 
 {% block content %}
+{% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
I'd say no, given that we're not translating this page (and I'm not sure if we should at the moment)